pkg/sync/sync_context_test.go

```diff
@@ -1,9 +1,12 @@
 package sync
 
 import (
+	"context"
 	"encoding/json"
 	"errors"
 	"fmt"
+	errors2 "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
Just a nit: Could we please rename the import to `apierrors` or something more obvious than `errors2`?

Reply:
yes, it was autogenerated, sry